main.tf

```
@@ -105,6 +105,25 @@ resource "aws_opensearch_domain" "this" {
       warm_enabled                  = try(cluster_config.value.warm_enabled, null)
       warm_type                     = try(cluster_config.value.warm_type, null)
 
+      dynamic "node_options" {
+        for_each = try(cluster_config.value.node_options, [])
+
+        content {
+
+          dynamic "node_config" {
+            for_each = try([node_options.value.node_config], [])
+
+            content {
+              count   = try(node_config.value.count, null)
+              enabled = try(node_config.value.enabled, null)
```

saves one less line when a config is passed
```suggestion
              enabled = try(node_config.value.enabled, true)
```